### PR TITLE
Minor: Created core-secret.yml

### LINKED ISSUE #45: Create k8s secret 'core-secret' for security
The secret will contain only one key - token.
The values of token will be automatically generated via sha1 cypher from current timestamp.

### PR DESCRIPTION
**Pull request by Oleksandr6676**
Created core-secret.yml for storing token value
Closes #45

### CODE REVIEW

File: kubernetes/core-secret.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: core-secret
+type: Opaque
+data:
+    token: token_placeholder
```
Comment (JanMate) [inline kubernetes/core-secret.yml:7]: I know you didn't meet `sed` bash tool so far, but the text string 'token_placeholder' is tough to read by machine. 

Better way is to set it like `<TOKEN_PLACEHOLDER>` or even better `%TOKEN_PLACEHOLDER%` as I mentioned in #45 (https://learnk8s.io/templating-yaml-with-code#using-templates-with-search-and-replace)

It helps the machine to parse the file and don't replace occasional wrong text string for example in metadata anytime.

**Review by JanMate — CHANGES_REQUESTED** [review]

File: kubernetes/core-secret.yml (hunk 1 above)
Comment (Oleksandr6676) [reply kubernetes/core-secret.yml:7]: Hm, I have just realized that I can only add encoded values here. If I add <TOKEN> or %TOKEN% it simply does not allow me to create this secret. But if I encode this value lt actually works fine. I can for example add encoded value here and then decode it inside our shell script to use with sed command but I'm not sure if this is optimal solution.

Comment (Oleksandr6676) [reply kubernetes/core-secret.yml:7]: I will check documentation, maybe we can use some other type of secret

Comment (JanMate) [reply kubernetes/core-secret.yml:7]: Great catch! You rocks! 👌🏽 🚀 

The best solution is to have a 'final' folder that will contain final generated yaml manifests for kubectl and this file will be only as a template one. 

What do you think about it? Is it meaningful and feasible?

Comment (Oleksandr6676) [reply kubernetes/core-secret.yml]: Hi Jan! Thanks for the explanation! Yes, that sounds good and reasonable. As I can see you have already changed value to <CORE_TOKEN>. Can we now push this PR?

Comment (JanMate) [reply kubernetes/core-secret.yml]: Thanks a lot for reminder. 👍🏽 Of course, we can merge it. I'll just do it. 👌🏽

**Review by JanMate — APPROVED** [review]

**Pull request merged**